evaluation/data/ims/README.md

```diff
@@ -0,0 +1,35 @@
+This is a  directory to regrid 1km and 4km IMS snow cover into various FV3 grid configurations. It includes several sub-directories below:
+
```

Comment:
can you add some clarity between the differences between 1 and 2? I assume 1) creates the index files to go from the IMS grid to the FV3 tiles while 2) creates the index files to go from the IMS grid to the 1D unstructured grid

Reply:
actually I guess this applies to 1, 2, 3 and 4; you don't need to do all four steps only one of them?

Reply:
Added as suggested.